databroker-cli/src/kuksa_cli.rs

```diff
@@ -36,13 +36,13 @@ const TIMEOUT: Duration = Duration::from_millis(500);
 const CLI_COMMANDS: &[(&str, &str, &str)] = &[
     ("connect", "[URI]", "Connect to server"),
     ("get", "<PATH> [[PATH] ...]", "Get signal value(s)"),
-    ("set", "<PATH> <VALUE>", "Set actuator signal"),
+    ("actuate", "<PATH> <VALUE>", "Set actuator signal"),
     (
         "subscribe",
         "<QUERY>",
         "Subscribe to signals with QUERY, if you use kuksa feature comma separated list",
     ),
-    ("feed", "<PATH> <VALUE>", "Publish signal value"),
+    ("publish", "<PATH> <VALUE>", "Publish signal value"),
```

Comment:
provide?

Reply:
Yes, that crossed my mind as well. I'm not against it, but my idea is that `publish` is that act of publishing a single signal value, while "provide" might be better suited for describing the act of continuously providing a signal. "publish" would then be the logical counterpart to "subscribe".

I suppose we will have a discussion around this in the API design rounds. I just couldn't stand having `feed` still there :smiling_face_with_tear: